tests/framework/CodeInterfaceTests/Instant/IAEA2D/sample/1/alias.xml

```diff
@@ -0,0 +1,22 @@
+<Materials>
```

Comment:
Is IAEA2D/sample/1 a generated directory (as in should it be in the repository)?

Reply:
I fixxed and removed all the un-useful files there (sorry for this...good catch)